edb/edgeql/compiler/stmtctx.py

```diff
@@ -153,6 +153,11 @@ def fini_expression(
             view_own_pointers = view.get_pointers(ctx.env.schema)
             for vptr in view_own_pointers.objects(ctx.env.schema):
                 _elide_derived_ancestors(vptr, ctx=ctx)
+                ctx.env.schema = vptr.set_field_value(
+                    ctx.env.schema,
+                    'is_from_alias',
```

Comment:
This field shouldn't be necessary,  There is already an `is_derived` field that already serves the same purpose.